src/components/RichText.tsx

```diff
@@ -1,34 +1,20 @@
-import React from 'react'
-import {TextStyle} from 'react-native'
-import {AppBskyRichtextFacet, RichText as RichTextAPI} from '@atproto/api'
-import {msg} from '@lingui/macro'
-import {useLingui} from '@lingui/react'
-import {useNavigation} from '@react-navigation/native'
+import React, { Fragment, useCallback, useMemo, useState } from 'react';
```

Comment:
Did an auto reformatter get used on the entire file? It is very hard to see what the actual differences are.